### ISSUE TITLE
Anomaly entities are invisible when on a holding platform

### ISSUE THREAD

**Issue by RogueNull** [SmashPhil/Vehicle-Framework#161]
**Describe the bug**
Anomaly entities are invisible when on a holding platform.

The following error can be seen in the logs:
```
Exception drawing 'HoldingPlatform9099' at cell (63, 0, 63):
System.NullReferenceException: Object reference not set to an instance of an object
[Ref D834447] Duplicate stacktrace, see ref for original
UnityEngine.StackTraceUtility:ExtractStackTrace ()
(wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition:Verse.Log.Error_Patch5 (string)
PerformanceFish.Log/Config/<>c:<.cctor>b__12_2 (string)
PerformanceFish.Log:Error (FisheryLib.Pools.PooledStringHandler)
PerformanceFish.Rendering.DynamicDrawManagerPatches/DrawDynamicThingsPatch/Logging:DrawException (Verse.Thing,System.Exception)
PerformanceFish.Rendering.DynamicDrawManagerPatches/DrawDynamicThingsPatch:DrawThingsNow ()
Verse.DynamicDrawManager:DrawDynamicThings ()
Verse.Map:MapUpdate ()
(wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition:Verse.Game.UpdatePlay_Patch1 (Verse.Game)
Verse.Root_Play:Update ()
```

This seems like a potential regression of #147.

**To Reproduce**
1. Start a new world with `Dev quicktest`
2. Enable god mode
3. Build 7x7 with a holding platform in the center (see screenshot below)
4. Use `Spawn pawn` to spawn an entity (`Toughspike` in this case, but others work too)
5. Use `Set anomaly level` to `Stirring`
6. Order a colonist to capture the entity
7. The entity will stop rendering once captured on the platform
8. See error

**Screenshots**
![wtvLM44](https://github.com/SmashPhil/Vehicle-Framework/assets/35264274/377df0a0-87d3-4bb2-89e4-72ec83358e15)

**Additional context**
Logs: https://gist.github.com/HugsLibRecordKeeper/cdf1c92967d6914d4bd90349a4b56676

**Comment by RogueNull** [comment]
Actually it looks like I was mistaken. This seems to be caused by Vanilla Expanded Framework.

Vehicle Framework alone is fine: ![image](https://github.com/SmashPhil/Vehicle-Framework/assets/35264274/0ebd369f-2579-452f-98bd-66589ae0b78a)

Logs: https://gist.github.com/HugsLibRecordKeeper/ae8b4bba2389377af2ce69e73a492a9a

Sorry about that. I thought Vanilla Expanded Framework was a dependency but got suspicious when I saw this mod hadn't been updated recently (and I know it worked last night).

Closing and opening under Vanilla Expanded Framework instead.
